onnx/inliner/inliner.cc

```diff
@@ -283,6 +267,8 @@ class InliningRenamer : private MutableVisitor {
     renamer.Bind<true>(*callee.mutable_output(), callnode.output());
 
     renamer.VisitFunction(&callee);
+    for (auto& v : *callee.mutable_value_info())
+      renamer.LookupOrRename(*v.mutable_name(), false);
```

Comment:
what happens for function inputs and outputs? Will they be renamed/bind to callnode inputs and outputs?